applications/AlgoCash.md

```diff
@@ -0,0 +1,163 @@
+
+Further distribution of Algo Shares are given to liquidity providers (e.g. ALS-DOT pair, ALS-aUSD pair, etc.) or ecosystem contributors. A total of 500,000 Algo Shares are distributed over a period of 1 year, and an equal amount of tokens are distributed per day.
+
+Advantages
+
+1. Funds in the crowd sale will not be abused, but used to establish the liquidity market.
+
+2. Regardless of the funds raised, sufficient selling liquidity can be provided to facilitate large amounts of funds to enter the market.
+
+3. Compared to joint curve issuance: a fair start, avoiding scientists from rushing
+
+4. Compared to AMM 2 pool mining: inflation tokens are not required to pay liquidity rents, and tokens are distributed to investors instead of those who "dig, sell and withdraw".
+
+5. Compared to auctions: Crowd sale is compatible with auction functions, but it is not just a simple fundraising. After the crowd sale is over, a market with abundant liquidity will be established immediately. Even having not raise enough funds, a market with sufficient selling liquidity can still be established, which AMM cannot do.
+
+
+### Ecosystem Fit 
+Acala, aUSD is generated in a collateral way.
+
+Basis Cash (on Ethereum), shares are initialized by 'AMM + 2nd pool' which would cause dramatic infaltion, forcing Yeild Farmers to sell their assets to the second market.
+
+## Team 
+
+### Team members
+* Nick Hsu, Blockchain Expert and Team Leader
+* Gang Chan, Full Stack Developer	
+
+### Contact
+* **Contact Name:** Nick Hsu
+* **Contact Email:** nick_hsu@sina.com
+
+
+### Legal Structure 
+No legal Entity
+
+### Team's experience
+Nick has 16 years of software development experience and 5 years working in Blochain area as developer and product owner.
+
+Gang is now working as a freelancer. He is a full stack developer proficient in C++, Rust, React and Solidity. He has 3 years software development experience and 2 years smart contract development experience.
+
+### Team Code Repos
+* https://github.com/ReserveLabs/AlgoCash
+
+### Team LinkedIn Profiles
+no
+
+
+## Development Roadmap 
+### Overview
+* **Total Estimated Duration:** 6 weeks
+* **Full-time equivalent (FTE):**  1.5
+* **Total Costs:** 5,000 DAI
+
+### Milestone 1 Example — Implement Substrate Modules 
+* **Estimated Duration:** 7 weeks
+* **FTE:**  2
+* **Costs:** 5,000 DAI  
+
+| Number | Deliverable | Specification |
+| ------------- | ------------- | ------------- |
+| 0 | License | Apache 2.0 / MIT / Unlicense |
+| 1 | Documentation | Specification of the background, components and working mechanism|
+| 2 | Smart Contract | AlgoCash smart contract repo. The smart contract can be deployed to any substrate chain with ink! pallet.|
```

Comment:
Could you specify the functions of the smart contract here a little bit better? These are basically the requirements of our contracts.

Reply:
The specification of smart contract has been added as a section